assets/src/components/admin/admin_add_modal.tsx

```diff
@@ -42,52 +36,47 @@ const fields = [
   { key: "device_id", label: "Device ID", FormCell: FormTextCell },
 ];
 
+const commonV2Params = {
+  header: { stop_id: "" },
```

Comment:
`stop_name` is the only header type supported by all V2 screens. Busway in particular doesn't support `stop_id`.

Reply:
Hmm, thanks for catching that. Pre-Fare is actually the opposite and doesn't support `stop_name`. I may just need to apply these individually to match what the app accepts.